concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Heggarty-2017-200	Heggarty, Paul and Anderson, Cormac and Scarborough, Matthew	2017		200	basic	English	Indo-European languages	http://cobl.clld.org	Heggarty2017		This is the basic list of 200 items which were selected for the Cognates in Basic Lexicon project. The compilers selected the items following various different criteria, such as ease of elicitation, non-fuzziness of the meaning, representation in Indo-European languages, and more. The numbers are at times higher than 200. This results from the earlier selection which containted more concepts which were no longer used in the official version.		
```

Comment:
I'm not sure, whether the domain `cobl.info` will be used for the clld app. If so, we'll have to make sure to keep `cobl.clld.org` as alias.

Reply:
So should I add cobl.info now, or should I rather leave it as is?

Reply:
I guess leave as is. Was rather a note-to-self.